.circleci/config.yml

```diff
@@ -214,13 +221,16 @@ workflows:
       - install_dependencies:
           requires:
             - checkout_code
-      - generate_ts_declarations:
+      - lint:
           requires:
             - install_dependencies
+      - generate_ts_declarations:
+          requires:
+            - lint
       - test_browsers:
           name: "test-Mac.Safari"
           requires:
-            - install_dependencies
+            - lint
```

Comment:
We can leave it to `install_dependencies` here imho. It should be enough, if `generate_ts_declarations` gets linted.

Reply:
Ok, that means that generating the ts declarations a will run parallell to the `gulp lint` / continue to run even if the `gulp lint`  fails - or I can add the linting as the last step of install_dependencies. Is that ok?

Reply:
I though that we have:
```
checkout->dependencies->lint->lint-dts
                     \->Safari->Chrome->Firefox->Edge->IE
```

Reply:
Changed it now as discussed :)

Reply:
Thank you! :)
Looks good. 👍